File: JeffSebastian/playground/src/main/java/nl/uva/se/ast/expression/Minus.java
package nl.uva.se.ast.expression;

import nl.uva.se.visitor.Visitor;

public class Minus extends Binary{

	public Minus(int lineNumber, int offset, Expression left, Expression right) {
		super(lineNumber, offset, left, right);
	}

	@Override
	public void accept(Visitor visitor) {
	visitor.visit(this);
	}
	

}

File: fabien/src/QL/operators/Minus.py

from NumericOperand import NumericOperand

class Minus(NumericOperand):
    def evaluate(self, answers):
        return self.left.evaluate(answers) - \
               self.right.evaluate(answers)

    def __repr__(self):
        return "%s - %s" % (self.left, self.right)
